src/Compilers/Core/Portable/DiagnosticAnalyzer/DiagnosticAnalysisContext.cs

```diff
@@ -1151,10 +1151,15 @@ public struct OperationAnalysisContext
         /// </summary>
         public ISymbol ContainingSymbol => _containingSymbol;
 
+        /// <summary>
+        /// <see cref="CodeAnalysis.SemanticModel"/> that can provide semantic information about the <see cref="IOperation"/>.
+        /// </summary>
+        public SemanticModel SemanticModel => _semanticModel;
```

Comment:
>public SemanticModel SemanticModel => _semanticModel; [](start = 8, length = 53)

It would be good to see where we can take advantage of this API. Otherwise, it is not clear how useful it is, what does it make better.